cookbook-react/src/recipes/collapsibles/valparaiso/Collapsible/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactNode, useState, useRef, useEffect } from 'react';
+import cn from 'classnames';
+
+import styles from './styles.module.scss';
+
+interface CollapsibleProps {
+  children: ReactNode;
+  title: string;
+  className?: string;
+  onChange?: (isOpen: boolean) => void;
+  defaultIsClosed?: boolean;
+  collapsibleId: string;
+}
+
+const useCollapse = (defaultIsOpen: boolean, onCollapse?: (isOpen: boolean) => void) => {
```

Comment:
just asking, could this be placed on a `hooks` folder? :P

Reply:
Yes, in the next PR I did that because I used it in another component. Can we leave it like this in this PR and check that change in https://github.com/Wolox/frontend-cookbook/pull/132 ?